src/App.js

```diff
@@ -1,29 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import './scss/App.scss';
+import React, { Component } from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
 
-export default App;
+  render() {
+    return (
+      <>
+        <BrowserRouter>
+          <Calculator />
+          <Routes>
+            <Route path="/calculator" element={<Calculator />} />
+          </Routes>
+        </BrowserRouter>
+      </>
+    );
+  }
+}
```

Comment:
- [ ] As listed in your requirements, you are to use functional components. Kindly refactor this component to a functional one instead.